ownCloud/Messages/MessageGroupCell.swift

```diff
@@ -0,0 +1,248 @@
+					NSLayoutConstraint.activate([
+						applyAllSwitch.topAnchor.constraint(equalTo: applyAllContainer.topAnchor, constant: 10),
+						applyAllSwitch.leftAnchor.constraint(equalTo: applyAllContainer.leftAnchor, constant: 20),
+						applyAllSwitch.bottomAnchor.constraint(equalTo: applyAllContainer.bottomAnchor, constant: -10),
+
+						applyAllLabel.leftAnchor.constraint(equalTo: applyAllSwitch.rightAnchor, constant: 10),
+						applyAllLabel.centerYAnchor.constraint(equalTo: applyAllSwitch.centerYAnchor),
+
+						showAllLabel.leftAnchor.constraint(greaterThanOrEqualTo: applyAllLabel.rightAnchor, constant: 20),
+						showAllLabel.rightAnchor.constraint(equalTo: applyAllContainer.rightAnchor, constant: -20),
+						showAllLabel.centerYAnchor.constraint(equalTo: applyAllSwitch.centerYAnchor)
+					])
+				}
+
+				self.applyThemeCollection(theme: Theme.shared, collection: Theme.shared.activeCollection, event: .initial)
+			}
+
+			if let alertView = alertView, let containerView = containerView {
+				self.addSubview(containerView)
+
+				containerView.addSubview(alertView)
+
+				if multiMessage, let applyAllContainer = applyAllContainer, let badgeLabel = badgeLabel {
+					containerView.addSubview(applyAllContainer)
+					containerView.addSubview(badgeLabel)
+				}
+
+				NSLayoutConstraint.activate([
+					containerView.leftAnchor.constraint(equalTo: self.leftAnchor, constant: 20),
+					containerView.rightAnchor.constraint(equalTo: self.rightAnchor, constant: -20),
+					containerView.topAnchor.constraint(equalTo: self.topAnchor, constant: 20),
+					containerView.bottomAnchor.constraint(equalTo: self.bottomAnchor, constant: noBottomSpacing ? 0 : -20),
+
+					alertView.leftAnchor.constraint(equalTo: containerView.leftAnchor),
+					alertView.rightAnchor.constraint(equalTo: containerView.rightAnchor),
+					alertView.topAnchor.constraint(equalTo: containerView.topAnchor)
+				])
+
+				if multiMessage, let applyAllContainer = applyAllContainer, let badgeLabel = badgeLabel {
+					NSLayoutConstraint.activate([
+						alertView.bottomAnchor.constraint(equalTo: applyAllContainer.topAnchor),
+						applyAllContainer.bottomAnchor.constraint(equalTo: containerView.bottomAnchor),
+						applyAllContainer.leftAnchor.constraint(equalTo: containerView.leftAnchor),
+						applyAllContainer.rightAnchor.constraint(equalTo: containerView.rightAnchor),
+
+						badgeLabel.centerYAnchor.constraint(equalTo: alertView.titleLabel.centerYAnchor),
+						badgeLabel.rightAnchor.constraint(equalTo: alertView.titleLabel.rightAnchor)
+					])
+				} else {
+					NSLayoutConstraint.activate([
+						alertView.bottomAnchor.constraint(equalTo: containerView.bottomAnchor)
+					])
+				}
+			}
+		}
+
+		self.accessoryType = .none
+	}
+
+	@objc func applyAllSwitchChanged() {
+		badgeLabel?.labelText = "+ \((messageGroup?.messages.count ?? 0) - 1) more"
```

Comment:
`localize` is missing (please check strings for localization)

Reply:
Thanks! Fixed.